Netable/Example/MockNetworkingService/JSONData/graphqlPosts.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Had to go ahead and add a second post json file because I changed the original to showcase Lossy, and it doesn't work/cooperate with Graphql from what I can tell!